dbms/src/Columns/ColumnString.h

```diff
@@ -250,6 +250,28 @@ class ColumnString final : public COWPtrHelper<IColumn, ColumnString>
         offsets.push_back(new_size);
     }
 
+    template <bool add_terminating_zero>
+    ALWAYS_INLINE inline void batchInsertDataImpl(size_t num, const char * pos, size_t length)
+    {
+        size_t old_size = chars.size();
+        size_t appended_size = 0;
+        const size_t single_str_size = length + (add_terminating_zero ? 1 : 0);
+
+        chars.resize(old_size + (single_str_size * num));
+        if (offsets.capacity() < offsets.size() + num)
```

Comment:
don't need to check it here because `offsets.reserve()` will do the same check

Reply:
> don't need to check it here because `offsets.reserve()` will do the same check

if is deleted